azurelinuxagent/common/event.py

```diff
@@ -277,6 +279,7 @@ def _add_event(self, duration, evt_type, is_internal, is_success, message, name,
         event.parameters.append(TelemetryEventParam('Message', message))
         event.parameters.append(TelemetryEventParam('Duration', duration))
         event.parameters.append(TelemetryEventParam('ExtensionType', evt_type))
+        event.parameters.append(TelemetryEventParam('ContainerId', os.environ.get(CONTAINER_ID, None)))
```

Comment:
a better default would be "UNINITIALIZED", or something like that. we don't want to be confused by values in Kusto

Reply:
Good point. Will update.